plugins/GSdx/GSLinuxDialog.cpp

```diff
@@ -9,7 +9,6 @@
  *
  *  This Program is distributed in the hope that it will be useful,
  *  but WITHOUT ANY WARRANTY; without even the implied warranty of
```

Comment:
@Gareth422, I believe you accidentally deleted a line from this file's license text. Ya might want to fix that.